### PR TITLE
Add Role delete functionality

### PR DESCRIPTION
**Pull request by DimalChandrasiri**
## Purpose
This PR will add Role delete functionality to the role listing.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <groupId>org.wso2.identity.apps</groupId>
     <artifactId>identity-apps</artifactId>
     <packaging>pom</packaging>
-    <version>1.0.50-SNAPSHOT</version>
+    <version>1.0.51-SNAPSHOT</version>
```
Comment (thanujalk) [inline pom.xml]: Let's remove this edit

File: apps/admin-portal/pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.wso2.identity.apps</groupId>
         <artifactId>identity-apps</artifactId>
-        <version>1.0.50-SNAPSHOT</version>
+        <version>1.0.51-SNAPSHOT</version>
```
Comment (thanujalk) [inline apps/admin-portal/pom.xml]: Let's remove this edit

File: pom.xml (hunk 1 above)
Comment (thanujalk) [reply pom.xml]: Please do the same for all other places

Comment (jeradrutnam) [reply pom.xml]: No more valid. As the PR is fixed now.

File: apps/admin-portal/pom.xml (hunk 2 above)
Comment (jeradrutnam) [reply apps/admin-portal/pom.xml]: No more valid. As the PR is fixed now.

File: pom.xml (hunk 1 above)
Comment (DimalChandrasiri) [reply pom.xml]: Fixed with PR cleanup

File: apps/admin-portal/pom.xml (hunk 2 above)
Comment (DimalChandrasiri) [reply apps/admin-portal/pom.xml]: Fixed with PR cleanup

**Comment by DimalChandrasiri** [comment]
> Please fix the PR. Commits are wrong.

Cleaned up the PR as requested.

File: apps/admin-portal/src/configs/app.ts [hunk 3]
```diff
@@ -57,6 +58,7 @@ export const ServiceResourcesEndpoint: ServiceResourcesType = {
     logout: `${GlobalConfig.serverHost}/oidc/logout`,
     me: `${GlobalConfig.serverHost}/scim2/Me`, // TODO: Remove this endpoint and use ID token to get the details
     profileSchemas: `${GlobalConfig.serverHost}/scim2/Schemas`,
+    permission: `${GlobalConfig.serverHost}/api/server/v1/permission-management/permissions`,
```
Comment (jeradrutnam) [inline apps/admin-portal/src/configs/app.ts]: Swap order. Arrange alphabetically.

File: apps/admin-portal/src/configs/app.ts [hunk 4]
```diff
@@ -33,6 +33,7 @@ interface ServiceResourcesType {
     logout: string;
     me: string;
     profileSchemas: string;
+    permission: string;
```
Comment (jeradrutnam) [inline apps/admin-portal/src/configs/app.ts]: Swap order. Arrange alphabetically.

File: apps/admin-portal/src/configs/app.ts (hunk 3 above)
Comment (DimalChandrasiri) [reply apps/admin-portal/src/configs/app.ts]: Fixed with commit f593963

File: apps/admin-portal/src/configs/app.ts (hunk 4 above)
Comment (DimalChandrasiri) [reply apps/admin-portal/src/configs/app.ts]: Fixed with commit f593963

**Pull request merged**